package.json

```diff
@@ -42,9 +44,6 @@
     "prettier": "^2.0.5",
     "ts-jest": "^26.3.0",
     "ts-node": "^9.0.0",
-    "tslint": "^6.1.3",
-    "tslint-config-prettier": "^1.18.0",
-    "tslint-plugin-prettier": "^2.3.0",
```

Comment:
So maybe is a good time to remove the example package 👉 @appnroll/template-lib after moved @appnroll/app-function-component and @appnroll/eslint-config ? 🤔 🙃

Reply:
0c189e217d7292443613ea74f5686901e0186543